ComputeClient/Compute.Client.UnitTests/Drs/ConsistencyGroupAccessorTests.cs

```diff
@@ -0,0 +1,300 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using DD.CBU.Compute.Api.Client;
+using DD.CBU.Compute.Api.Client.Drs;
+using DD.CBU.Compute.Api.Contracts.Drs;
+using DD.CBU.Compute.Api.Contracts.Requests;
+using DD.CBU.Compute.Api.Contracts.Requests.Drs;
+
+namespace Compute.Client.UnitTests.Drs
+{
+	[TestClass]
+	public class ConsistencyGroupAccessorTests : BaseApiClientTestFixture
+	{
+		[TestMethod]
+		public async Task GetConsistencyGroups_ReturnsResponse()
+		{
+			requestsAndResponses.Add(ApiUris.MyAccount, RequestFileResponseType.AsGoodResponse("GetMyAccountDetails.xml"));
+			requestsAndResponses.Add(ApiUris.GetConsistencyGroups(accountId), RequestFileResponseType.AsGoodResponse("GetConsistencyGroupsResponse.xml"));
+
+			var client = GetWebApiClient();
+			var accessor = new ConsistencyGroupAccessor(client);
+			var consistencyGroups = await accessor.GetConsistencyGroups();
+
+			Assert.IsNotNull(consistencyGroups);
+			Assert.AreEqual(3, consistencyGroups.Count());
+			Assert.AreEqual("GoodCGNoReIp", consistencyGroups.ElementAt(0).name);
+
+			// Check the progress item for the second CG
+			var consistencyGroupWithProgress = consistencyGroups.ElementAt(1);
+
+			// Check the progress object.
+			Assert.IsNotNull(consistencyGroupWithProgress.progress);
+			Assert.AreEqual("CREATE_CONSISTENCY_GROUP", consistencyGroupWithProgress.progress.action);
+
+			// Check the progress step mapping.
+			Assert.IsNotNull(consistencyGroupWithProgress.progress.step);
+			Assert.AreEqual(10, consistencyGroupWithProgress.progress.step.number);
+
+			// Check the serverpair mapping.
+			Assert.IsNotNull(consistencyGroupWithProgress.serverPair);
+			Assert.AreEqual(1, consistencyGroupWithProgress.serverPair.Count());
+
+			var serverPair = consistencyGroupWithProgress.serverPair.ElementAt(0);
+			Assert.AreEqual("2861363d-1f66-4bce-b568-82c4d3582ed7", serverPair.id);
+			Assert.IsNotNull(serverPair.sourceServer);
+			Assert.IsNotNull(serverPair.targetServer);
+			Assert.AreEqual("e55a4707-e7cd-4dad-9c03-ef834b6e4528", serverPair.sourceServer.id);
+			Assert.AreEqual("SourceServer1", serverPair.sourceServer.name);
+			Assert.AreEqual("f3174580-2d51-400f-9584-f0e32784e7e7", serverPair.targetServer.id);
+			Assert.AreEqual("TargetServer1", serverPair.targetServer.name);
+
+			// Check the ReIp Rules defined
+			var consistencyGroupWithReIp = consistencyGroups.ElementAt(2);
+			Assert.IsNotNull(consistencyGroupWithReIp.serverPair);
+			Assert.AreEqual(1, consistencyGroupWithReIp.serverPair.Count());
+			var serverPairWithReIp = consistencyGroupWithReIp.serverPair.ElementAt(0);
+			Assert.IsTrue(serverPairWithReIp.reIpAddressRulesDefined);
+		}
+
+		[TestMethod]
+		public async Task GetConsistencyGroups_PaginatedResponse()
+		{
+			PageableRequest pagingOptions = new DD.CBU.Compute.Api.Contracts.Requests.PageableRequest();
```

Comment:
very minor but think we can re-write this as 

```          
           var pagingOptions = new DD.CBU.Compute.Api.Contracts.Requests.PageableRequest
			{
				PageSize = 3,
				PageNumber = 1,
				Order = "sourceDataCenterId"
			};
```
Just a little bit more concise - reading around in c# think the steer is var should be used for local variables where its fairly obvious what the assignment will be based on the right side of the expression

Reply:
Yup, fixed both the pagingOptions and the var things.